src/PKSim.Presentation/Presenters/Protocols/ProtocolItemPresenter.cs

```diff
@@ -73,6 +88,69 @@ protected IParameter ParameterFrom(IParameterDTO parameterDTO)
          return parameterDTO.Parameter;
       }
 
+      public string DisplayFor(string containerName)
+      {
+         return _representationInfoRepository.DisplayNameFor(RepresentationObjectType.CONTAINER, containerName);
+      }
+      
+      protected void SetTargetOrgan(string organName, ISchemaItem schemaItem)
```

Comment:
Codes come 100% from SimpleprotocolPresenter with the addition of the schemaItem parameter that was before the simpleProtocol implicitely